File: homemarket-domain/src/main/java/com/plateer/homemarket/entity/Product.java
package com.plateer.homemarket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Product {
	
	private int productId;
	private String productName;
	private String productDetail;
	private String categories;
	private int quantity;
	private int price;
	private String memberId;
	private String uploadDate;
	

}

File: vue/src/model/Product.js
class Product{
    constructor(productId, productName, productDetail, categroies, quantity, price, memberId, uploadDate){
        this.productId = productId;
        this.productName = productName;
        this.productDetail = productDetail;
        this.categroies = categroies;
        this.quantity = quantity;
        this.price = price;
        this.memberId = memberId;
        this.uploadDate = uploadDate;
    }
}

export default Product;